lib/sax/parser_feedback_simulator.js

```diff
@@ -76,13 +89,31 @@ ParserFeedbackSimulator.prototype._ensureTokenizerMode = function (tn) {
 ParserFeedbackSimulator.prototype._handleStartTagToken = function (token) {
     var tn = token.tagName;
 
-    if (tn === $.SVG)
+    if (tn === $.SVG) {
         this._enterNamespace(NS.SVG);
+        foreignContent.adjustTokenSVGAttrs(token);
+        foreignContent.adjustTokenXMLAttrs(token);
+    }
 
     else if (tn === $.MATH)
         this._enterNamespace(NS.MATHML);
 
+    else if (tn === $.IMAGE)
+        token.tagName = $.IMG;
```

Comment:
Well, for our use case I need to be as close to browser behavior as possible, as I'm going to do various streaming transformations, so if browser treats `<image />` as `<img />` then I should be able to see it this way too, so that any transformations that are applicable to `img` will be applied to `image` as well, any selectors that look for `img`, should fine `image` too, and so on and so on.

As for the newlines - that might be debatable, but, personally, I'd prefer to be as consistent with the real handling as possible in order not to decrease risk of running into weird bugs in future.

With the "close to the source code as much as possible" you could argue that we can also disable NUL-character and CRLF normalizations as they might also be seen as non-critical (especially in JS land), but that's a way too big rabbit hole to get into IMO :)